docs/docs/guide/08-loan/07-cancel.md

```diff
@@ -0,0 +1,74 @@
+# Cancel a loan
+
+As a borrower, you have the option to cancel a loan you have created if you no
+longer want to proceed with it. However, this action is only possible if the
+loan's current status is marked as "requested".
+
+If you decide to cancel the loan, the collateral tokens that were being held as
+security for the loan will be transferred back to your account from the module
+account. This means that you will regain possession of the collateral tokens you
+had originally put up for the loan.
+
+```go title="x/loan/keeper/msg_server_cancel_loan.go"
+package keeper
+
+import (
+	"context"
+	"fmt"
```

Comment:
Not really important but the `fmt` import could be removed because it's not being used

Reply:
https://github.com/ignite/cli/pull/3352/commits/7aa3ad9844ea0930fd1a323b1d1a621dd4bf57be